### DISCUSSION TITLE
Wiki doesnt render the coursor line

### DISCUSSION THREAD

**Discussion by SirSeba98** [requarks/wiki#6796] [Error / Bug Report]
Hello, 

i just noticed that our wiki doesnt render the coursor line anymore.
The version that we are running is the newest 2.5.300.

If i open the editor and then change the browser tab and go back the cursor line gets displayed.

I tried this with 2 Browsers, Microsoft Edge and Google Chrome.

This happens in all editors.

**Comment by NGPixel** [comment]
Have you tried on different computers? from different users?

**Reply by SirSeba98** [reply]
Yes, we have tried from 3 different users on 3 different computers.
The same thing. When the editor is opened, the Cursor line doesnt get displayed.
If you switch to another tab and go back to wiki tab then the cursor line is shown.

Update: When I open the editor, close it and then open it again on the same page the cursor seems to appear.
I have also tried a new browser which is not chromium based and on that I dont have any issues.
The cursor line is displayed as soon as I open the editor.

As far as I can see, it has something to do with chromium.

**Reply by docdoceg** [reply]
Getting this as well on chromium based browsers, can't seem to replicate it reliably, feels random. But closing the editor and re-opening it does solve it.

**Comment by SirSeba98** [comment]
Does someone have a work around for this or know how to fix it?
I wanted to migrate our old wiki (mediawiki) to wikijs, but now due to this issue that at least in my case happens all the time.
I can't migrate it and take the new wiki live.
